### PR TITLE
disabled state added to create room button

### PR DESCRIPTION
**Pull request by mevlutcantuna**

### CODE REVIEW

File: gurubu-client/src/app/components/room/nickname-form.tsx [hunk 1]
```diff
@@ -18,18 +18,16 @@ const defaultNickname = () => {
...
 };
 
-
-
-
 // If roomId is provided, then the user is joining a room.
 const NicknameForm = ({ roomId }: IProps) => {
   const [nickname, setNickname] = useState(defaultNickname);
-  const [groomingType, setGroomingType] = useState<GroomingType>(GroomingType.PlanningPoker);
+  const [groomingType, setGroomingType] = useState<GroomingType>(
```
Comment (aahmetcakir) [inline gurubu-client/src/app/components/room/nickname-form.tsx:24]: can you convert indentation to as is?

File: gurubu-client/src/app/components/room/nickname-form.tsx [hunk 2]
```diff
@@ -52,7 +50,7 @@ const NicknameForm = ({ roomId }: IProps) => {
     const payload = {
       nickName: trimmedNickName,
       // TODO?: Actually, backend api should be accept string value for groomingType
-      groomingType: groomingType === 'PlanningPoker' ? "0" : "1"
+      groomingType: groomingType === "PlanningPoker" ? "0" : "1",
```
Comment (aahmetcakir) [inline gurubu-client/src/app/components/room/nickname-form.tsx:53]: can you convert indentation to as is

File: gurubu-client/src/app/components/room/nickname-form.tsx [hunk 3]
```diff
@@ -141,18 +138,16 @@ const NicknameForm = ({ roomId }: IProps) => {
...
-          priority
-          src="/logo.svg"
-          alt="logo"
-          width={30}
-          height={30}
-        />
-        GuruBu</h1>
+        <Image priority src="/logo.svg" alt="logo" width={30} height={30} />
+        GuruBu
+      </h1>
```
Comment (aahmetcakir) [inline gurubu-client/src/app/components/room/nickname-form.tsx:143]: same

File: gurubu-client/src/app/components/room/nickname-form.tsx [hunk 4]
```diff
@@ -141,18 +138,16 @@ const NicknameForm = ({ roomId }: IProps) => {
...
       <h1 className="nickname-form__title">Welcome to Gurubu</h1>
       <div className="nickname-form__action-wrapper">
         <div className="nickname-form__input-wrapper">
-          <label htmlFor="nickname-input" className="nickname-form__label-enter-room">
+          <label
+            htmlFor="nickname-input"
+            className="nickname-form__label-enter-room"
+          >
```
Comment (aahmetcakir) [inline gurubu-client/src/app/components/room/nickname-form.tsx:150]: same

**Comment by mertcanaltin** [comment]
@mevlutcantuna hello do you need support here, feel free to ping us please

**Pull request closed without merging**